docs/appendices/compliance.rst

```diff
@@ -1,18 +1,23 @@
 .. highlight:: psql
+
 .. _sql_supported_features:
 
 =======================
 SQL standard compliance
 =======================
 
-This section provides a list of features that CrateDB supports and to what
-extent it conforms to the current SQL standard `ISO/IEC 9075`_ "Database
-Language SQL".
+This page documents the standard SQL (`ISO/IEC 9075`_) features that
+CrateDB supports, along with implementation notes and any associated caveats.
+
+.. CAUTION::
+
+    This list may be incomplete or lacking information. Consult the primary
```

Comment:
I'm not sure whether _"incomplete or lacking information"_ conveys same message as _"...approximate and ...nonconforming in their implementation"_

just personal perception:

new version - some SQL features are missing (true), lacking information (true)

old version - I believe it meant "don't rely on your knowledge of SQL standards on existing feature, check main reference to check implementation differences if there are any"

perhaps would be nice to keep new version and also warn about unconformity

Reply:
agreed. I restored the original version with a copyedit